flow/designs/sky130hd/aes/constraint.sdc

```diff
@@ -1,15 +1,18 @@
-###############################################################################
-# Created by write_sdc
-# Fri May  7 11:36:37 2021
-###############################################################################
 current_design aes_cipher_top
-###############################################################################
-# Timing Constraints
-###############################################################################
-create_clock -name clk -period 3.7439 -waveform {0.0000 1.8720} [get_ports {clk}]
-###############################################################################
-# Environment
-###############################################################################
-###############################################################################
-# Design Rules
-###############################################################################
+
+set clk_name  clk
+set clk_port_name clk
+set clk_period 5.9 
```

Comment:
Why the shift from 3.7439 to 5.9?

Reply:
There two reasons behind this
1)  The set_ws for with the old clock was -1.59   -- I have tried to make all of the setup_ws at post GR to be close to 0 or slightly negative to monitor improvements
2)  All of these designs did not have the IO constraints and hence the timing reports were not looking at the full picture. Now they do have IO constraints